src/modules/roc_fec/target_openfec/roc_fec/of_encoder.h

```diff
@@ -52,6 +52,9 @@ class OFEncoder : public IEncoder, public core::NonCopyable<> {
     //! Get buffer alignment requirement.
     virtual size_t alignment() const;
 
+    //! Get the maximum value of a source block for a used scheme.
+    virtual size_t max_block_length() const;
```

Comment:
Comment is outdated.

Reply:
s/for a used scheme/for a scheme being used/

Reply:
Done